ci/common.sh

```diff
@@ -180,6 +180,8 @@ linux_get_dependencies () {
     apt update
     apt install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl libpcre3 libpcre3-dev libldap2-dev
     apt-get install -y libyaml-dev
+    wget https://github.com/libexpat/libexpat/releases/download/R_2_5_0/expat-2.5.0.tar.gz
```

Comment:
What is the purpose of installing this software? I don't see where it is used.

Reply:
This is because luaexpact depends on Expat and requires a version greater than 2.4; otherwise, it will throw an error. The Expat package in the CLI environment is less than version 2.4. I used the SDK: https://luarocks.org/modules/kong/lua-resty-aws/1.5.1-1.

Reply:
![image](https://github.com/user-attachments/assets/7f35601b-df88-4fb7-b057-acccc445bafc)

Reply:
So we also need to change the APISIX docker package, right?

Reply:
Yes, if want to use the SDK.

Reply:
We have removed this dependency from the fork and it is no longer needed.